cipd_packages/ruby/ruby_ship/tools/ruby_ship_build.sh

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env bash
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -x
+#VERIFY PARAMETERS
+if [ $# -eq 0 ]
+  then
```

Comment:
nit: move this up a line and follow with ';' to make it consistent with the rest of the file.

Reply:
done!